libs/designsystem/src/lib/components/button/button.component.scss

```diff
@@ -149,16 +141,22 @@
     width: 100%;
   }
 
+  // Only apply focus ring if pointer device can hover
+  // (effectively desktop/mouse devices):
+  @include focus() {
+    --kirby-button-border-color: #{$focus-ring-color};
+  }
+
   &:hover,
   &:focus {
     opacity: 0.8;
   }
 
   &:disabled {
-    background-color: get-color('semi-light');
-    border-color: transparent;
-    pointer-events: none;
+    --kirby-button-background-color: #{get-color('semi-light')};
+    --kirby-button-border-color: transparent;
```

Comment:
Not sure about this one - it was intentionally overriding background & border-color instead of just the variables, to ensure disabled state always "wins". I know it works with the variables as well, but consider keeping this additional "safety-net" for future regressions (some rule with higher specificity redefening the background & border-color and/or a consumer settings these variables from the outside)

Reply:
I agree. I did not think about that.